Bottom: 1fcc11fbb8042fffeb49fad81683004dd403fe8e
Top:    dd7a2521a4feff30c3e497a236008db3a0c6fff6
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 13:14:12 -0500

Refresh of add-buildbot-role

---
diff --git a/roles/buildbot-master/tasks/main.yml b/roles/buildbot-master/tasks/main.yml
index 7164f3d0..138f692d 100644
--- a/roles/buildbot-master/tasks/main.yml
+++ b/roles/buildbot-master/tasks/main.yml
@@ -46,6 +46,6 @@
 
 - name: Add buildbot_master host
   add_host:
-    name: buildbot_master
+    name: '{{ container_hostname }}'
     pubkey: "{{ output.stdout }}"
     ip_addr: "{{ buildbot_ip_addr }}"
